test/WebJobs.Script.Tests/Rpc/WorkerConfigFactoryTests.cs

```diff
@@ -207,5 +216,36 @@ public void AddArgumentsFromAppSettings_JavaOpts(string expectedArgument, string
                 Assert.Equal(expectedArgument, workerDescription.Arguments[1]);
             }
         }
+
+        [Theory]
+        [InlineData("{FUNCTIONS_WORKER_RUNTIME_VERSION}/@os/@architecture", "3.7/LINUX/X64")]
+        [InlineData("{FUNCTIONS_WORKER_RUNTIME_VERSION}/@architecture", "3.7/X64")]
+        [InlineData("{FUNCTIONS_WORKER_RUNTIME_VERSION}/@os", "3.7/LINUX")]
```

Comment:
can you please share documenation on `@` notation? also, is it a common pattern to refer environment variables  using braces `{}`?

Reply:
I took the `{}` cue from our default templates where we wrap app settings in braces (e.g. `{AzureWebJobsStorage}`). The `@` symbol was based on my own thinking for what might be a reasonable choice 😛. I thought about how we tag people in emails with `@` and that resolves to their actual email alias. Happy to replace with something else if you have suggestions.

Reply:
From the patterns used in function.json: https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-expressions-patterns, 
`%apsettinname%` and `{varaible name}`

Reply:
Okay, I can make that change.